gate-web/config/gate.yml

```diff
@@ -58,6 +58,9 @@ services:
 redis:
   connection: redis://localhost:6379
 
+slack:
+  baseUrl: https://slack.com/api
+
```

Comment:
Can you ensure gate starts up normally even without this config. Just so we don't break folks in OSS

Reply:
I added another `@ConditionalOnProperty` to check for the url as well

Reply:
Speaking from lots of experience, it's best to make sure that gate starts up locally without this property as well as with this property. I've broken OSS several times because I forgot to check that :)

Reply:
Confirmed that gate OSS runs without any slack config :)